src/super4pcs/super4pcs.cc

```diff
@@ -1367,29 +1374,23 @@ bool MatchSuper4PCSImpl::ComputeRigidTransformation(const vector< pair<Point3D,
 // Performs N RANSAC iterations and compute the best transformation. Also,
 // transforms the set Q by this optimal transformation.
 bool MatchSuper4PCSImpl::Perform_N_steps(int n, cv::Mat* transformation,
-                                    std::vector<Point3D>* Q) {
+                                         std::vector<Point3D>* Q, bool enable_timer) {
   if (transformation == NULL || Q == NULL) return false;
 
-#ifdef TEST_GLOBAL_TIMINGS
-    Timer t (true);
-#endif
-
   float last_best_LCP = best_LCP_;
   bool ok;
-  int64 t0 = clock();
+
   for (int i = current_trial_; i < current_trial_ + n; ++i) {
-    ok = TryOneBase();
-
-    float fraction_try =
-        static_cast<float>(i) / static_cast<float>(number_of_trials_);
-    float fraction_time = static_cast<float>(clock() - t0) / CLOCKS_PER_SEC /
-                          options_.max_time_seconds;
-    float fraction = max(fraction_time, fraction_try);
-    printf("done: %d%c best: %f                  \r",
-           static_cast<int>(fraction * 100), '%', best_LCP_);
-    fflush(stdout);
+    ok = TryOneBase(enable_timer);
+
+    float fraction_trials = i / static_cast<float>(number_of_trials_);
+    //float fraction = max(fraction_time, fraction_try);
```

Comment:
Please revert these changes in the main branch: printing is useful to follows exploration progress.(you can comment them in your own brach if you wich.